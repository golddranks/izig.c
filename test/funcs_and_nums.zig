const std = @import("std");

fn hoge(piyo: []const u8) void {
    const stdout = std.io.getStdOut().writer();
    stdout.print("こんにちは、{s}！\n", .{piyo}) catch {};
}

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    hoge("fuga");
    try stdout.print("Nice: {d}\n", .{42 + 69});
}
