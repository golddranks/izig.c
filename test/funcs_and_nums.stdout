こんにちは、fuga！
Nice: 111
